tests/unit/discovery/windows/test_windows.py

```diff
@@ -0,0 +1,33 @@
+import sys
+
+import pytest
+
+from virtualenv.discovery.py_spec import PythonSpec
+
+
+@pytest.mark.skipif(sys.platform != "win32", reason="no Windows registry")
```

Comment:
Why we need to skip if we're going to mock the registry?

Reply:
Because before the mocks get applied, we will need to import `winreg` first (through `virtualenv.discover.windows.pep514`). This import itself will fail on non-Windows.